test/nat.cooltt

```diff
@@ -1,96 +1,7 @@
-def symm : {
-  (A : univ) (x : A) (y : A) (p : Id A x y)
-  → Id A y x
-} = {
-  \A x y p =>
-  elim p @ {u v _ => Id A v u} [
-  | refl => refl
-  ]
-}
-
-def trans : {
-  (A : univ) (x : A) (y : A) (z : A)
-  (p : Id A x y) (q : Id A y z)
-  → Id A x z
-} = {
-  \A x y z p =>
-  elim p @ {u v _ => (q : Id A v z) → Id A u z} [
-  | refl => \q => q
-  ]
-}
-
-def trans-assoc : {
-  (A : univ) (s : A) (t : A) (u : A) (v : A)
-  (p : Id A s t) (q : Id A t u) (r : Id A u v)
-  → Id
-      {Id A s v}
-      {trans A s u v {trans A s t u p q} r}
-      {trans A s t v p {trans A t u v q r}}
-} = {
-  \A s t u v p =>
-  elim p @ {s t p =>
-    (q : Id A t u) (r : Id A u v)
-    → Id
-        {Id A s v}
-        {trans A s u v {trans A s t u p q} r}
-        {trans A s t v p {trans A t u v q r}}
-  } [
-  | refl => \q r => refl
-  ]
-}
-
-def trans-left-unit : {
-  (A : univ) (x : A) (y : A) (p : Id A x y)
-  → Id {Id A x y} {trans A x x y refl p} p
-} = {
-  \A x y p => refl
-}
-
-def trans-right-unit : {
-  (A : univ) (x : A) (y : A) (p : Id A x y)
-  → Id {Id A x y} {trans A x y y p refl} p
-} = {
-  \A x y p =>
-  elim p @ {x y p => Id {Id A x y} {trans A x y y p refl} p} [
-  | refl => refl
-  ]
-}
-
-def trans-symm-refl : {
-  (A : univ) (x : A) (y : A) (p : Id A x y)
-  → Id {Id A x x} {trans A x y x p {symm A x y p}} refl
-} = {
-  \A x y p =>
-  elim p @ {x y p =>
-    Id {Id A x x} {trans A x y x p {symm A x y p}} refl
-  } [
-  | refl => refl
-  ]
-}
-
-def cong : {
```

Comment:
Some of this stuff is already in `test/nat-path.cooltt` using `path` instead of `Id`, but not all of it. We should now be able to port them to the path type, which would also be a good test of the new auto-hcom (and ordinary hcom).

Reply:
yeah i was thinking i might take a crack at that, per your comment yesterday even